src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,78 @@
+.card {
+  width: 198px;
+  @include border(1px, solid, $border-color, 5px);
+  .card__item {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 32px 16px 16px;
+    .item__image-container {
+      display: flex;
+      align-items: center;
+      align-self: center;
+      margin-bottom: 40px;
+      .item__image {
+        width: 160px;
+        height: 134px;
+      }
+    }
+    .item__name {
+      @include font(500, 12px, 18px, $font-color, Roboto);
+      margin-bottom: 4px;
+    }
+    .item__code {
+      @include font(400, 10px, 14px, $font-color--secondary, Roboto);
+      margin-bottom: 16px;
+    }
+    .item__rate-container.rate {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: flex-end;
+      width: 100%;
+      margin-bottom: 24px;
+      .rate__review.review {
+        .review__item {
+          @include font(400, 10px, 14px, $font-color, Roboto);
+        }
+      }
+
+    }
+    .item__price-container.price {
+      display: flex;
+      justify-content: space-between;
+      width: 100%;
+      align-items: center;
+      margin-bottom: 16px;
+      .price__text {
+        @include font(400, 12px, 18px, $font-color--secondary, Roboto);
+      }
+      .price__value {
+        @include font(700, 16px, 18px, $font-color, Roboto);
+      }
+    }
+    .item__button {
+      height: 40px;
```

Comment:
Setting fixed widths and heights is generally a bad idea. Width of the element can be determined by it's parent and set to 100%. Height could be determined by line-height + paddings.

Reply:
> Setting fixed widths and heights is generally a bad idea. Width of the element can be determined by it's parent and set to 100%. Height could be determined by line-height + paddings.

I was change properties for better flexibility